### PR TITLE
MP 3.3 RC1

### PR DESCRIPTION
**Pull request by jclingan**
Please review. Updated FAQ as well. Feel free to make suggestions.

### CODE REVIEW

**Comment by jclingan** [comment]
Yeah, @xstefank, that is intended . Basically,  I try to get everything as near to "done" as possible, but since the README shows up front-and-center, I want to make it clear that  this is still a RC release.

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 1]
```diff
@@ -58,7 +99,7 @@ The Maven coordinates for this Eclipse release are as follows:
 <dependency>
     <groupId>org.eclipse.microprofile</groupId>
     <artifactId>microprofile</artifactId>
-    <version>3.2</version>
+    <version>3.3</version>
```
Comment (kwsutter) [inline spec/src/main/asciidoc/architecture.asciidoc]: Why did you change the version for the MP 3.2 description?  That doesn't look right...

Comment (jclingan) [reply spec/src/main/asciidoc/architecture.asciidoc]: Fixed.
